types/src/share.rs

```diff
@@ -18,6 +19,8 @@ mod info_byte;
 
 pub use info_byte::InfoByte;
 
+const FIRST_SPARSE_SHARE_SEQUENCE_LENGTH_OFFSET: usize = NS_SIZE + appconsts::SHARE_INFO_BYTES;
```

Comment:
```suggestion
const SHARE_SEQUENCE_LENGTH_OFFSET: usize = NS_SIZE + appconsts::SHARE_INFO_BYTES;
```
it doesn't depend on being sparse or compact